CHANGELOG.md

```diff
@@ -3,11 +3,116 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 1.0.0
 
-### Minimal Go version to build OPA: 1.22
+> **_NOTES:_**
+>
+> * The minimum version of Go required to build the OPA module is **1.22**
+
+We are excited to announce **OPA 1.0**, a milestone release consolidating an improved developer experience for the future of Policy as Code.
+The release makes new functionality designed to simplify policy writing and improve the language's consistency the default.
+
+### Changes to Rego in OPA 1.0
+
+Below we highlight some key changes to the defaults in OPA 1.0:
+
+- The `if` & `contains` keywords are now mandatory when declaring rule heads, not just when using the `rego.v1` import.
```

Comment:
```suggestion
- Using `if` for all rule definitions and `contains` for multi-value rules is now mandatory, not just when using the `rego.v1` import.
```

It could be misunderstood such that "contains" was mandatory for all rule heads, I think? Trying to clarify...